modules/comments/mmd.json

```diff
@@ -9,14 +9,57 @@
     "userOwnable": false,
     "trackable": true,
     "taggable": true,
-    "peerReplicable": false,
+    "peerReplicable": true,
```

Comment:
is this change intentional?

Reply:
I am doing this changes from MSSP system hence it is coming like this